ui/src/pages/exercisePage.tsx

```diff
@@ -128,6 +156,10 @@ export const ExercisePage = () => {
         setExcercises(response);
       })
       .catch((err) => console.error(err));
+      getUserFavorites('123').then((data) => {
```

Comment:
here, the token could be used to find the username on the backend and then find the favorites